save-format.md

```diff
@@ -561,11 +557,163 @@ eb: byte
 49a: int  - Hard Wins
 49e: int  - Special Wins
 
+4a2: int  - Sum of Score
```

Comment:
This is 32 bit? seems possible to cap out if so :X

Reply:
I'll take a look at what happens when I set this to a very high value. but 2^31-1 is still pretty large

Reply:
Good catch, its a signed 64 bit number, I'll update this field and also add the type to the "Types" section